dialogue/TownNPCs/Mocha/InitialInteractions/CardinalPistachio/CardinalPistachio_Mocha_First3.json

```diff
@@ -28,8 +28,8 @@
     "pl": [
       "O, tylko podoba mi się ta nazwa, przez tego że, ty wiesz, wszyscy macie inne kolory.",
```

Comment:
I think "O, tak was po prostu lubię nazywać, przez to, że wszyscy macie inne kolory" will sound a bit more natural